dashboard/src/pages/repos/new.tsx

```diff
@@ -0,0 +1,282 @@
+    color: vars.text.black1,
+  },
+})
+
+// copy from /pages/apps
+// and delete unnecessary styles
+const ContentContainer = styled('div', {
+  base: {
+    marginTop: '24px',
+  },
+})
+
+// copy from /pages/apps/new
+const InputFormContainer = styled('div', {
+  base: {
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '20px',
+
+    background: vars.bg.white3,
+    border: `1px solid ${vars.bg.white4}`,
+    borderRadius: '4px',
+    padding: '8px 12px',
+  },
+})
+const InputForm = styled('div', {
+  base: {},
+})
+const InputFormText = styled('div', {
+  base: {
+    fontSize: '16px',
+    alignItems: 'center',
+    fontWeight: 700,
+    color: vars.text.black1,
+
+    marginBottom: '4px',
+  },
+})
+const InputBar = styled('input', {
+  base: {
+    padding: '8px 12px',
+    borderRadius: '4px',
+    border: `1px solid ${vars.bg.white4}`,
+    fontSize: '14px',
+    marginLeft: '4px',
+
+    width: '320px',
+
+    display: 'flex',
+    flexDirection: 'column',
+
+    '::placeholder': {
+      color: vars.text.black3,
+    },
+  },
+})
+
+interface FormProps {
+  label: string
+  type?: JSX.InputHTMLAttributes<HTMLInputElement>['type']
+  placeholder?: JSX.InputHTMLAttributes<HTMLInputElement>['placeholder']
+  value: JSX.InputHTMLAttributes<HTMLInputElement>['value']
+  onInput: JSX.InputHTMLAttributes<HTMLInputElement>['onInput']
+}
+
+const Form = (props: FormProps): JSXElement => {
+  return (
+    <InputForm>
+      <InputFormText>{props.label}</InputFormText>
+      <InputBar
+        type={props.type ?? 'text'}
+        placeholder={props.placeholder ?? ''}
+        value={props.value}
+        onInput={props.onInput}
+      />
+    </InputForm>
+  )
+}
+
+const ToggleButtonsWrapper = styled('div', {
+  base: {
+    display: 'flex',
+    flexDirection: 'row',
+  },
+})
+
+interface ToggleButtonsProps<T> {
+  items: {
+    value: T
+    label: string
+  }[]
+  selected: T
+  onChange: (value: T) => void
+}
+
+const ToggleButtons = <T extends unknown>(props: ToggleButtonsProps<T>): JSXElement => {
+  return (
+    <ToggleButtonsWrapper>
+      <For each={props.items}>
+        {(item) => (
+          <Button color='black1' size='large' onclick={() => props.onChange(item.value)}>
+            {item.label}
+          </Button>
+        )}
+      </For>
+    </ToggleButtonsWrapper>
+  )
+}
+
+const SshDetails = styled('div', {
+  base: {
+    color: vars.text.black2,
+    marginBottom: '4px',
+  },
+})
+
+const PublicKeyCode = styled('code', {
+  base: {
+    display: 'block',
+    padding: '8px 12px',
+    fontFamily: 'monospace',
+    fontSize: '14px',
+    background: vars.bg.white2,
+    color: vars.text.black1,
+    border: `1px solid ${vars.bg.white4}`,
+    borderRadius: '4px',
+  },
+})
+
+const SystemPublicKey = (): JSXElement => {
+  const [systemPublicKey] = createResource(() => client.getSystemPublicKey({}))
+
+  return (
+    <div>
+      <SshDetails>
+        公開鍵を入力せずにSSH認証でリポジトリを登録する場合、以下のSSH公開鍵が認証に使用されます。
```

Comment:
ユーザーにNeoShowcaseに入力させるのは、ユーザーが生成した鍵ペアのうち秘密鍵側ですね
公開鍵側はユーザーが自身で自分のリポジトリにデプロイキーなどとして登録します

秘密鍵の欄を空にしてNeoShowcaseにリポジトリを登録する場合は、NeoShowcaseはシステムのデフォルトの鍵を使うので、getSystemPublicKeyで得られる公開鍵をユーザーが自分のリポジトリにデプロイキーとして登録する必要がある、という仕組みです
```suggestion
        秘密鍵を入力せずにSSH認証でリポジトリを登録する場合、以下のSSH公開鍵が認証に使用されます。
```

Reply:
勘違いしていました、レビューありがとうございます🙇

a23f731 にて修正しました